test/issue1645-dflags-build.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# If DFLAGS are not processed, dub for library would fail
+DFLAGS="-w" $DUB --root="$CURR_DIR"/1-staticLib-simple --build=plain
+if DFLAGS="-asfdsf" $DUB --root="$CURR_DIR"/1-staticLib-simple --build=plain 2>/dev/null; then
+  echo "Should not accept this DFLAGS"
+fi
```

Comment:
This isn't testing anything. It will always pass. You need to use at least `set -e`

Reply:
Fixed.